build.gradle

```diff
@@ -277,7 +302,7 @@ project('kcbq-confluent') {
     configurations.all {
         resolutionStrategy {
             // depending on debezium results in us pulling in the most recent kafka version, but we want to use 1.1.1.
```

Comment:
need to update this comment / or figure out if we can stick with 1.1.1.

Reply:
@avocader Can I update the comment?